### ISSUE TITLE
[Windows] Crash if included directory does not exist

### ISSUE THREAD

**Issue by itsMapleLeaf** [facebook/flow#2438]
If a directory under [include] in .flowconfig doesn't exist, the binary sort of just starts freaks out without any sort of coherent error message. It just repeats this over and over.

```
Kingdaro@Kingdaro-PC MINGW64 ~/Documents/Code/javascript/fchat-next (dev) $ flow
Launching Flow server for C:\Users\Kingdaro\Documents\Code\javascript\fchat-next
Spawned flow server (pid=12160)
Logs will go to C:\Users\Kingdaro\AppData\Local\Temp\flow\CzCzBUserszBKingdarozBDocumentszBCodezBjavascriptzBfchat-next.log
Started a new flow server: -flow is still initializing; this can take some time. [parsing] \flow is still initializing; this can take some time. [parsing] |flow is still initializing; this can take some time. [parsing] /flow is still initializing; this can take some time. [local inference] -The flow server is not responding (3 retries remaining): [processing] \Launching Flow server for C:\Users\Kingdaro\Documents\Code\javascript\fchat-next
Spawned flow server (pid=9480)
Logs will go to C:\Users\Kingdaro\AppData\Local\Temp\flow\CzCzBUserszBKingdarozBDocumentszBCodezBjavascriptzBfchat-next.log
Started a new flow server: |flow is still initializing; this can take some time. [parsing] /
```

**Comment by mrkev** [comment]
Can @kagawagao or anyone confirm if this still happens?

**Comment by kagawagao** [comment]
@mrkev sorry for this, I have no Windows device by hand 😂

**Comment by kearnspadraig** [comment]
Had a similar error. Two of us working, one on a Mac, one on Windows.
A directory was listed as "/flow-typed/npm/", causing constant crashes on windows, but no issue on the Mac.

This issue got us by it.

**Comment by mroch** [comment]
this was likely fixed by b19e64992816ec55326f874014e823a0554e4f76
